framework/mesh/unpartitioned_mesh/unpartitioned_mesh.h

```diff
@@ -59,10 +59,10 @@ class UnpartitionedMesh
   unsigned int Dimension() const { return dim_; }
   void SetDimension(unsigned int dim) { dim_ = dim; }
 
-  const BoundBox& BoundingBox() const { return bound_box_; }
+  //  const BoundBox& BoundingBox() const { return bound_box_; }
   void ComputeBoundingBox();
 
-  void SetMeshOptions(const Options& opts) { mesh_options_ = opts; }
+  //  void SetMeshOptions(const Options& opts) { mesh_options_ = opts; }
```

Comment:
If you are removing these, then remove them. Do not comment out API.

`BoundingBox()` seems like something that might be useful?

Reply:
`ComputeBoundingBox()` is actually needed in  framework/mesh/io/vtk_io.cc (line 443). But `const BoundBox& BoundingBox() const { return bound_box_; }` was never used (isn't it a getter for the private var?). Can you tell me what may be going on here?

Reply:
> But `const BoundBox& BoundingBox() const { return bound_box_; }` was never used (isn't it a getter for the private var?)

There is nothing wrong with that.